src/core/default/commands/function/logs.py

```diff
@@ -34,21 +35,25 @@ def add_arguments(self, parser: ArgumentParser) -> None:
         )
         parser.add_argument(
             "--number",
-            action="store_true",
```

Comment:
I agree that this could be renamed to `limit`. Could you also set the default value via the argument parser using the `const` and `default` parameters? Right now it is just set to 10,000 via line 83 (`limit=number_val if number_val else 10000,`). 

https://stackoverflow.com/questions/15301147/python-argparse-default-value-or-specified-value